src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/repository/PrisonerRepository.kt

```diff
@@ -19,14 +19,14 @@ import uk.gov.justice.digital.hmpps.prisonersearchindexer.model.SyncIndex
 @Repository
 class PrisonerRepository(
   private val client: RestHighLevelClient,
-  private val restTemplate: OpenSearchRestTemplate,
+  private val openSearchRestTemplate: ElasticsearchOperations,
```

Comment:
annoyingly the opensearch configuration doesn't define a bean of type `OpenSearchRestTemplate`, so have to use the parent class instead.

Reply:
So you'd have to cast to use any new open search support? How strange.